src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,35 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.health = health;
```

Comment:
```suggestion
  constructor(name) {
    this.health = 100;
```
Looks much cleaner like this and you don't have to pass health to each class.